h2/stream.py

```diff
@@ -1040,18 +1040,26 @@ def _build_hdr_validation_flags(self, events):
             is_trailer = isinstance(
                 events[0], (_TrailersSent, TrailersReceived)
             )
+            is_response_header = isinstance(
+                events[0], (_ResponseHeadersSent, ResponseReceived)
```

Comment:
(Trivial) Seeing these side-by-side, I wonder if we should drop `Headers` from the name of this event, i.e., `_ResponseSent`. Better symmetry, and matches what I did for `_TrailerSent`.